src/components/Builder/BasicInfo/index.jsx

```diff
@@ -145,8 +152,16 @@ const BasicInfo = ({ profileData }) => {
               {
                 pattern: WHOLE_NO_VALIDATOR,
                 message: "Experience must be a whole number",
-              }
-            ]}           
+              },
+              {
```

Comment:
remove this validator, we dont want any validation on this field

Reply:
Done!